Makefile

```diff
@@ -12,5 +15,5 @@ steps_are_complete:
 	./util/"steps are complete.sh"
 
 verify_complete:
-	$(MAKE) -C exercises/complete verify
+	$(MAKE) -C exercises/complete
```

Comment:
I don't think we should change the verify to do everything?
(I think that is what you are doing.

Reply:
I've just made it such that we verify everything about the complete snapshot inside the container, so we also verify the container works and has all the dependencies working at the same time. This is now implicitly `make all` so it verifies, executes, and (in the next PR) does Duvet as well.

I just left the steps verification outside the container because that's not something participants will do. So `tmx` doesn't need to be in the container for e.g.

Reply:
`assert DafnyCodeIsBeingVerified() by` https://github.com/aws-samples/how-to-write-more-correct-software-workshop/actions/runs/3191365615/jobs/5207587005#step:3:1747